src/routes/profile/+page.server.js

```diff
@@ -9,3 +10,11 @@ export const load = async (serverLoadEvent) => {
         user: response
     }
 };
+
+export const actions = {
+    default: async ({cookies}) => {
+        if(softDeleteUser(pool, cookies.get('uuid'))) {
```

Comment:
Il manque un `await` lors de l'appel de ta fonction.
```suggestion
        if(await softDeleteUser(pool, cookies.get('uuid'))) {
```